Readme.md

```diff
@@ -3,6 +3,8 @@
 
  X-Response-Time middleware for koa.
 
+___Notice: `koa-response-time@2` supports `koa@2`; if you want to use this module with `koa@1`, please use `koa-response-time@1`.___
```

Comment:
hmm yea I guess we're kinda screwed since  we didnt use it proactively, going back and publishing new 1.x versions for each middleware would suck, might as well start I suppose